mp2p_icp_filters/src/FilterDecimateRangeAdaptive.cpp

```
@@ -0,0 +1,261 @@
+{
+    mrpt::system::COutputLogger::setLoggerName("FilterDecimateRangeAdaptive");
+}
+
+void FilterDecimateRangeAdaptive::initialize_filter(const mrpt::containers::yaml& c)
+{
+    MRPT_START
+
+    MRPT_LOG_DEBUG_STREAM("Loading these params:\n" << c);
+    params.load_from_yaml(c);
+
+    MRPT_END
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If MRPT_END is a macro then please configure it.

(unknownMacro)